src/source/mod.rs

```diff
@@ -351,6 +351,23 @@ where
     {
         blt::high_pass(self, freq)
     }
+
+    fn low_pass_with_q(self, freq: u32, q: f32) -> BltFilter<Self>
+    where
+        Self: Sized,
+        Self: Source<Item = f32>,
+    {
+        blt::low_pass_with_q(self, freq, q)
+    }
+
```

Comment:
Lets add a comment explaining what q is like you have in blt.rs

Reply:
Added comments similar to that of blt.rs